1080i/SkinSettings.xml

```diff
@@ -798,12 +806,12 @@
                         <onclick>Skin.ToggleSetting(DisableClock)</onclick>
                     </control>
 
-                    <control type="radiobutton" id="19709" description="Clearlogo">
+                    <control type="radiobutton" id="19710" description="Show clearart">
                         <visible>Integer.IsEqual(Window.Property(CurrentFocus),9007)</visible>
                         <include>Defs_Settings_Button</include>
-                        <label>$LOCALIZE[31170]</label>
-                        <selected>Skin.HasSetting(EnableClearlogo)</selected>
-                        <onclick>Skin.ToggleSetting(EnableClearlogo)</onclick>
+                        <label>$LOCALIZE[31434]</label>
+                        <selected>Skin.HasSetting(ShowOSDClearArt)</selected>
+                        <onclick>Skin.ToggleSetting(ShowOSDClearArt)</onclick>
```

Comment:
Add an "enable" condition so that the setting can't be enabled if "ShowInfoPaused" is on.
`<enable>!Skin.HasSetting(ShowInfoPaused)</enable>`